src/idsk/components/related-content/template.njk

```diff
@@ -0,0 +1,14 @@
+
+<div class="govuk-grid-column-{%- if params.oneThird %}one-third{% else %}two-thirds{% endif %}">
+  <div class="idsk-related-content{%- if params.classes %} {{ params.classes }}{% endif %}" data-module="idsk-related-content">
```

Comment:
toto by som prepisal radsej tak {{ params.classes if params.classes }} je to citatelnejsie